### PR TITLE
updated TiltingEquivalence

### PR DESCRIPTION
**Pull request by mohamed-barakat**

### CODE REVIEW

File: DerivedCategories/examples/julia/TiltingEquivalence.jl [hunk 1]
```diff
@@ -1,133 +1,167 @@
...
 
 ℚ = HomalgFieldOfRationals()
 
-q = RightQuiver( "q",
+q = RightQuiver( "q_𝓞",
           [ "𝓞(0)", "𝓞(1)", "𝓞(2)" ],
           [ "x0", "x1", "x2", "y0", "y1", "y2" ],
           [ 1, 1, 1, 2, 2, 2 ],
           [ 2, 2, 2, 3, 3, 3 ] )
+SetLabelsAsLaTeXStrings( q, julia_to_gap([ g"x_1", g"x_2", g"x_3", g"y_0", g"y_1", g"y_2" ]) )
```
Comment (kamalsaleh) [inline DerivedCategories/examples/julia/TiltingEquivalence.jl:15]: Only one comment, maybe setting these LaTeX attributes would be improved after merging https://github.com/homalg-project/Algebroids/pull/65 and requiring the latest version of Algebroids package.

Comment (mohamed-barakat) [reply DerivedCategories/examples/julia/TiltingEquivalence.jl:15]: Now merged.

Comment (mohamed-barakat) [reply DerivedCategories/examples/julia/TiltingEquivalence.jl:15]: Shall I update the PR?

Comment (kamalsaleh) [reply DerivedCategories/examples/julia/TiltingEquivalence.jl:15]: Yes please

**Comment by kamalsaleh** [comment]
Thank you very much :)

**Comment by mohamed-barakat** [comment]
I thank you.

**Pull request merged**